docs/content/faq.md

```diff
@@ -144,5 +144,39 @@ The Azure mixins are going to be full fledged tools that communicate with the
+ 
+You never have to use mixins, other than the built-in exec. You could write a custom dockerfile and then call a bash script. But most people find that working with mixins is easier.
+
+## Are Porter Bundles and Docker-Compose the same?
+
+Porter is never a replacement for an existing tool. Think of it as doing extra nice things on top of what those great tools already do!
+ * Packages everything you use to deploy in a single artifact that can be easily distributed over registries and across air gapped networks.
+ * Bundles can be signed and the signature verified before installing the bundle to improve supply chain security.
+ * Makes for more reliable deployments because it has the exact version of a CLI that goes with the deployment. End users don’t need to install the right versions of tools locally (sometimes different apps require different versions).
+ * Provides metadata about a deployment that you can use in many ways. End users can run porter explain to quickly see how to customize an installation with parameters, what credentials are needed by the bundle. No need to read separate installation docs to figure out how to deploy an app.
+ * Reduces the operational knowledge required to manage an application. Often a deployment uses multiple tools, for example terraform, AND helm, AND kubectl and they are glued together with bash scripts. When those are put inside a bundle, it simplifies how much the end user needs to know. It’s always a single command, porter upgrade . In many cases the end user doesn’t even need to know what tools are used, or how to use those tools.
+ * Makes it easier for a team to manage an application securely. Porter remembers the actions performed previously on the bundle, the parameter values used previously, the current version of the bundle, etc. 
+ * You can put together the necessary parameters and credentials so your teammates don’t need to each hunt down the proper values themselves, or even worse copy sensitive credentials into local environment variables on every machine that does the deployment. Secrets stay in a secret store and aren’t copied around.
+Those are just some of the reasons why working with bundles with the tools embedded inside is helpful vs using the same tools standalone.
+
+## Where can I use Porter in Day to Day Projects?
+
+ * Create a bundle for a side project. It handles setting up any infrastructure, such as creating a VM or other cloud resources, and it then also deploys your app. Maybe pushing your serverless function as one example, or running a helm chart. 
+ * This is useful because often we figure out how to do these things while working on the project and then after not having time for the project most people forget how to deploy their project. That makes it super hard to start back up on the project! So using bundles helps make the best use of your limited side project time
```

Comment:
This originally was a single paragraph in the Slack comment. This doesn't stand alone as an independent list item.
```suggestion
 * Create a bundle for a side project. It handles setting up any infrastructure, such as creating a VM or other cloud resources, and it then also deploys your app. Maybe pushing your serverless function as one example, or running a helm chart. 

   This is useful because often we figure out how to do these things while working on the project and then after not having time for the project most people forget how to deploy their project. That makes it super hard to start back up on the project! So using bundles helps make the best use of your limited side project time
```